README.md

```
@@ -1,138 +1,288 @@
-    }}
-    dismissDuration={500}
-    disableTapToDismiss
-    side="right"
-    presetAnimation="fadeIn"
-    backgroundColor="black"
-    borderRadius={12}
-  >
-    <Tooltip.Text text="Some copy..." style={{ color: "#000", fontSize: 16 }} />
-  </Tooltip.Content>
-</Tooltip.Root>;
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)